src/views/Dashboard.vue

```diff
@@ -142,7 +135,7 @@ export default {
             // only keep the unread ones with specific reasons
             return notifications.filter((n) => {
                 return (n.unread && ['assign', 'mention', 'review_requested'].includes(n.reason))
-            })
+            }).slice(0, 7)
```

Comment:
Why so few? I thought, as the payload is pretty small and a medium amount of notifications is hidden anyway, why not getting and showing more?